### ISSUE TITLE
React-Native link does not work because package name does not correspond with gradle import

### ISSUE THREAD

**Issue by ThaJay** [zmxv/react-native-sound#88]
```js
dependencies {
    compile project(':react-native-sound')
    compile fileTree(dir: "libs", include: ["*.jar"])
    compile "com.android.support:appcompat-v7:23.0.1"
    compile "com.facebook.react:react-native:+"  // From node_modules
}
```
sould contain
```js
    compile project(':RNSound')
```
Mildly annoying as all the other ones I use get this right.

**Comment by benvium** [comment]
PR #105 addresses this issue

**Comment by benvium** [comment]
`react-native link react-native-sound` works. I've updated the manual install steps to perform the same changes the link command does.

**Comment by paapii3d** [comment]
React-native link react-native-sound only links my IOS, it doesn't link my android

```
/projects/hcdd$:react-native link react-native-sound
Scanning folders for symlinks in /projects/hcdd/node_modules (11ms)
rnpm-install info iOS module react-native-sound is already linked
``` 

so wen i run `react-native run-android` i get an error. How can i fix this? But IOS works fine

**Comment by trepidity** [comment]
@paapii3d what version of RNS are you running? I released 0.10.9 that should not have this issue.
